src/ECER.Clients.RegistryPortal/ecer.clients.registryportal.client/src/components/SpecializedCertificationOptions.vue

```diff
@@ -0,0 +1,95 @@
+<template>
+  <ECEHeader title="Specialized certification options" />
+  <div class="d-flex flex-column ga-3 my-6">
+    <p>There are 2 types of specialized certifications in B.C. that you can add to your ECE Five Year certificate.</p>
+    <ol class="ml-10">
+      <li>Infant and Toddler Educator (ITE)</li>
+      <ul class="ml-10" style="list-style-type: disc">
+        <li>You must have completed an infant and toddler educator training program</li>
+      </ul>
+
+      <li>Special Needs Educator (SNE)</li>
+      <ul class="ml-10" style="list-style-type: disc">
+        <li>You must have completed a special needs early childhood educator training program</li>
+      </ul>
+    </ol>
+    <p>
+      You’ll need to request an official transcript from your educational institution. It must be sent directly from the educational institute to the ECE
+      Registry.
+    </p>
+    <p class="mt-6">What do you want to add to your certificate?</p>
+    <v-form ref="specializationForm" validate-on="lazy">
+      <v-checkbox
+        v-model="selected"
+        class="ml-n2"
+        value="Ite"
+        color="primary"
+        label="Add Infant and Toddler Educator (ITE)"
+        hide-details="auto"
+        @update:model-value="handleSelection"
+      ></v-checkbox>
+
+      <v-checkbox
+        v-model="selected"
+        class="ml-n2"
+        value="Sne"
+        color="primary"
+        label="Add Special Needs Educator (SNE)"
+        hide-details="auto"
+        @update:model-value="handleSelection"
+      ></v-checkbox>
+      <v-input
+        v-if="applicationStore.draftApplication.certificationTypes?.length == 0"
+        auto-hide="auto"
+        :model-value="selected"
+        :rules="[hasSelection]"
+      ></v-input>
```

Comment:
confused with the v-if on this component and the rules. 

We only check if the user has no certification type....

Reply:
Yes, it's convoluted but we don't need a selection if the user has 5 Year and is being asked about adding Ite/Sne... But in the case of Specialization selection we need at least one to be selected.